db/migrations/2022083100430000-add-isTransactional-to-tokens.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
+  async up (queryInterface, Sequelize) {
+    await queryInterface
+    .addColumn("tokens", "isTransactional", {
```

Comment:
I think that would be better to have `isTransactional` column in the `network_tokens` table, because a token can be used as transactional in one network and as reward in other. By moving the column we don't need to duplicate token information in the database.

Do you agree @moshmage ?